pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -136,6 +140,9 @@ cdef dict timedelta_abbrevs = {
 
 _no_input = object()
 
+TIMEDELTA_MIN_NS = np.iinfo(np.int64).min + 1
+TIMEDELTA_MAX_NS = np.iinfo(np.int64).max
```

Comment:
if these are only used inside raising blocks, no need to define them here

Reply:
They're used again at the bottom of the module, when defining `Timedelta.{min,max}`. I thought that _might_ be slightly clearer, explicitly linking the Timedelta boundaries to int sizes. But it's preferred to refer to these constants by some other, more canonical name (`sys.maxsize`?), happy to do that.

Reply:
the canonical names would probably be `util.INT64_MAX` and `NPY_NAT+1`